src/blocks/content-helper/content-helper-error.tsx

```diff
@@ -52,14 +66,58 @@ export class ContentHelperError extends Error {
 		Object.setPrototypeOf( this, ContentHelperError.prototype );
 	}
 
-	public ProcessedMessage( className = '' ): JSX.Element {
-		// Errors that need to display the "Contact Us" message.
-		const contactUsErrorCodes = [
+	/**
+	 * Returns an error message element that can contain HTML.
+	 *
+	 * Warning: Any HTML passed to this function should be sanitized.
```

Comment:
Since we force `wp_kses_post` in `get_credentials_not_set_message()`, I do not see a problem with the current approach, unless we get careless in the future.

When researching, the only method I found to sanitize in the JS function is using third-party libraries (for examlpe DOMPurify), and I wanted to avoid it if possible. Is this the approach you're suggesting?